packages/react-scripts/scripts/eject.js

```diff
@@ -94,9 +94,7 @@ prompt(
   delete appPackage.scripts['eject'];
   Object.keys(appPackage.scripts).forEach(function (key) {
     appPackage.scripts[key] = appPackage.scripts[key]
-      .replace(
-        new RegExp(ownPackageName + ' (\\w+)', 'g'),
-        'node scripts/$1.js'
+      .replace(/react-scripts (\w+)/g, 'node scripts/$1.js'
       );
```

Comment:
Style nit: since the line is shorter now, let's bring this closing paren and the semicolon to the end of the previous line?